docs/UserGuide.md

```diff
@@ -30,6 +38,12 @@ title: User Guide
 
 ## Quickstart
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source:** Please refer to the [Introduction to UI](#appendix-a---introduction-to-the-ui) if you're  trying TBM out for the first  time.
```

Comment:
Theres an extra space between first and time, and also between you're and trying